src/components/Sidebar/components/Backdrop/Backdrop.js

```diff
@@ -0,0 +1,33 @@
+import PropTypes from 'prop-types';
+import styled, { css } from 'react-emotion';
+
+const baseStyles = ({ theme, visible }) => css`
+  label: sidebar-backdrop;
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  background-color: black;
```

Comment:
I would think the designers might have picked something like `theme.colors.n900` instead.